groups/committee-oversight/groups.yaml

```diff
@@ -5,6 +5,7 @@ groups:
     description: |-
       Technical Oversight Committee Email
     settings:
+      AllowExternalMembers: "false"
       WhoCanPostMessage: "ANYONE_CAN_POST"
       ReconcileMembers: "true"
     owners:
```

Comment:
Can you make all the _current_ TOC members OWNERS?

Note that we're currently have personal accounts; add the following for me:

`evan.k.anderson@gmail.com`
`evana@vmware.com`

I'll let @dprotaso @n3wscott @rhuss decide what email they want on here, rather than outing any email addresses they want to keep private.

Reply:
Fixed, moved current toc members owners and the others(emeritus) to members.

Reply:
`dprotaso@gmail.com` for me please

Reply:
`n3wscott@chainguard.dev` for me please!

Reply:
rhuss@redhat.com please.

Reply:
Cool, I'll push a commit tomorrow with all the emails submitted before 12:00 UTC